html/browsers/windows/support-open-cross-origin.html

```diff
@@ -0,0 +1,3 @@
+<meta charset=utf-8>
+<p>Follow this link to open a new browsing context in a separate origin. 
+<a target=second href="http://www2.{{location[host]}}/html/browsers/windows/support-open-cross-origin.html">link</a>.
```

Comment:
If you want substitution to work you need to have `.sub.html` as filename suffix.